aiidalab_widgets_base/structures.py

```diff
@@ -215,143 +241,160 @@ def get_example_structures(examples):
 
     def _on_select_structure(self, change):  # pylint: disable=unused-argument
         """When structure is selected."""
-        if not self._select_structure.value:
-            return
-        structure_ase = get_ase_from_file(self._select_structure.value)
-        self.on_structure_selection(structure_ase=structure_ase, name=self._select_structure.label)
+
+        self.structure = None if not self._select_structure.value else get_ase_from_file(self._select_structure.value)
```

Comment:
I'd prefer the positive equivalent:
```suggestion
        self.structure = get_ase_from_file(self._select_structure.value) if self._select_structure.value else None
```

Reply:
Ok.